pgaudi/main.py

```diff
@@ -14,12 +14,16 @@
 import os
 import yaml
 from functools import partial
+import sys
+import psutil
+import signal
 
 # Gaudi
 import gaudi.parse
 
 # Pgaudi
-from . import parallel, treatment, similarity, create_output
+# from . import parallel, treatment, similarity, create_output
+import parallel, treatment, similarity, create_output
```

Comment:
What happened here? Please use explicit imports.

Reply:
This is because when I modified the code and verified if everything worked correctly, I executed the code as a script with python main.py. In this way, I had an error with the imports, so for my checks I changed it. I forgot to undo the change for the commit.